### DISCUSSION TITLE
Feature Request: Package for Cloudron [updated: community provided]

### DISCUSSION THREAD

**Discussion by heliostatic** [go-shiori/shiori#330] [Feature requests] [answered]
After one day of using Shiori locally, I'm hooked. I think it would be a great addition to the Cloudron.io app selection, too.

Cloudron is "...a platform to run apps on your server and keep them up-to-date and secure." Basically a self-hosting platform with one-click app installs.

https://cloudron.io/documentation/developer/packaging/

**Comment by slfhstr** [comment, marked as answer]
Hi all

Just letting you know that I created a custom package of Shiori for Cloudron (https://cloudron.io) - a brilliant PaaS for selfhosting.

Cloudron have now added Shiori to their official App Store.

So if you want to deploy Shiori in an easy self-hosted platform, check out Cloudron (2 apps free).

**Reply by fmartingr** [reply]
Would you mind adding a section under the installation docs linking to the cloudron package to have better visibility of this?

**Reply by slfhstr** [reply]
It's listed in the Cloudron App Store.
https://www.cloudron.io/store/com.github.go_shiori.html
Too many apps to list them all under installation, if you mean Cloudron's installation docs.
Maybe you mean Shiori's installation docs ?
I don't have access for that.
